### PR TITLE
chore: fixes envoy config for static configuration

### LINKED ISSUE #6: Does wasm plugin support static configuration?
Currently I am trying to build an e2e for the filter (see https://github.com/jcchavezs/modsecurity-wasm-filter-e2e). Right now I am struggling to get rules to work declared statically (with no ruleserver, plain envoy), I tried this configuration

```
                http_filters:
                  - name: envoy.filters.http.wasm
                    typed_config:
                      "@type": type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
                      config:
                        name: "my_plugin"
                        root_id: "my_root_id"
                        configuration:
                          "@type": "type.googleapis.com/google.protobuf.StringValue"
                          value: |
                            SecDebugLogLevel 5
                            SecDebugLog modsec.log
                            SecRuleEngine On
                            SecRule REQUEST_URI "@streq /admin" "id:1,phase:1,t:lowercase,deny"
                        vm_config:
                          runtime: "envoy.wasm.runtime.v8"
                          vm_id: "my_vm_id"
                          configuration: {}
                          code:
                            local:
                              filename: ./bin/modsecurity-filter.wasm
```

and this configuration

```
                http_filters:
                  - name: envoy.filters.http.wasm
                    typed_config:
                      "@type": type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
                      config:
                        name: "my_plugin"
                        root_id: "my_root_id"
                        configuration:
                          "@type": "type.googleapis.com/google.protobuf.StringValue"
                          value: |
                            {
                              "rules_inline" : "
                                SecDebugLogLevel 5
                                SecDebugLog modsec.log
                                SecRuleEngine On
                                SecRule REQUEST_URI \"@streq /admin\" \"id:1,phase:1,t:lowercase,deny\"
                              "
                            }
                        vm_config:
                          runtime: "envoy.wasm.runtime.v8"
                          vm_id: "my_vm_id"
                          configuration: {}
                          code:
                            local:
                              filename: ./bin/modsecurity-filter.wasm
```

But I don't get to block the request. Is there anything I am doing wrong?

You can reproduce my e2e test following the instructions of this repository: https://github.com/jcchavezs/modsecurity-wasm-filter-e2e

### PR DESCRIPTION
**Pull request by M4tteoP**
Fix #6 

Fixes the envoy yaml file related to static configuration to test the wasm module just with envoy. 

Signed-off-by: Matteo Pace <pace.matteo96@gmail.com>

### CODE REVIEW

File: envoy/envoy_binary_static_configuration.yaml [hunk 1]
```diff
@@ -29,27 +29,21 @@ static_resources:
...
+                # root_id: "my_root_id"
+                # if your wasm filter requires custom configuration you can add as follows
                 configuration:
                   "@type": "type.googleapis.com/google.protobuf.StringValue"
                   value: |
-                    SecRuleEngine On
-                    SecRule ARGS:param1 "test" "id:1,phase:1,pass,status:200,msg:'Test rule'"
-                    SecRule ARGS:param1 "attack" "id:2,phase:1,deny,status:400,msg:'Test rule'"
+                    {
+                      "rules":"SecRuleEngine On\nSecRule ARGS:param1 \"test\" \"id:1,phase:1,pass,status:200,msg:'Test rule'\"\nSecRule ARGS:param1 \"attack\" \"id:2,phase:1,deny,status:400,msg:'Test rule'\""
```
Comment (M4tteoP) [inline envoy/envoy_binary_static_configuration.yaml:38]: Based on the [parsing code](https://github.com/intel/modsecurity-wasm-filter/blob/main/wasmplugin/envoy-wasm-modsecurity.cc#L135-L145), I came up with this JSON that is accepted by the WASM module. Open to any alternatives to make it more readable

**Comment by jcchavezs** [comment]
LGTM

**Review by jcchavezs — APPROVED** [review]

**Comment by jcchavezs** [comment]
We are already using this config in https://github.com/jcchavezs/modsecurity-wasm-filter-e2e and it works correctly.

**Comment by bryanwux** [comment]
Thank you for your contribution! @M4tteoP

**Pull request merged**